doc/en/user/source/community/oauth2/index.rst

```diff
@@ -306,6 +306,53 @@ From UI it is also possible to set the ``Response Mode`` value. The field can be
 
 Finally the admin can allow the sending of the client_secret during an access_token request trough the ``Send Client Secret in Token Request``. Some OpenId implementation requires it for the Authorization Code flow when the client app is a confidential client and can safely store the client_secret.
 
+OpenID Connect With Attached Access Bearer Tokens
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The OpenID Connect plugin allows the use of Attached Bearer Access Tokens. This is typically used by automated (i.e. desktop or external Web Service) to access the Geoserver REST API.
+
+The setup process is as follows:
+
+#. Setup your OAuth2 OpenID Connect configuration as normal
+#. On the OpenID Connect configuration screen (bottom), makes sure "Allow Attached Bearer Tokens" is checked
+#. You can not use ID Tokens as a Role Source for the attached Bearer Tokens (see below)
+
+To Use:
+
+#. Obtain an Access Token from the underlying IDP
+#. Attach the access token to your HTTP request headers
+
+`Authorization: Bearer <token>`
+
+The Access Token (JWT) is validated;
+
+#. The Access Token is used to get the "userinfo" endpoint.  The underlying IDP will verify the token (i.e. signature and expiry)
+#. The Audience of the Token is checked that it is the same as the Geoserver configured Client Id.  This make sure an Access Token for another application is not being inappropriately reused in Geoserver (cf. `AudienceAccessTokenValidator.java`).
```

Comment:
```suggestion
#. The Audience of the Token is checked that it is the same as the GeoServer configured Client Id.  This make sure an Access Token for another application is not being inappropriately reused in GeoServer (cf. `AudienceAccessTokenValidator.java`).
```

Reply:
We do not have the habit of referencing Java classes in our documentation; not saying it is a bad idea just that it has not been done before